pkgs/development/python-modules/srvlookup/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, dnspython
+, mock, nose
+}:
+
+buildPythonPackage rec {
+  pname = "srvlookup";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zf1v04zd5phabyqh0nhplr5a8vxskzfrzdh4akljnz1yk2n2a0b";
+  };
+
+  buildInputs = [ dnspython ];
```

python packages have to be propagated
```suggestion
  propagatedBuildInputs = [ dnspython ];
```

---

@jonringer thanks for spotting this. Seems I missed the notification for this comment. Fixed. PTAL.